src/Dropdown.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { theme } from 'styled-system'
+
+import { Box, Flex, Icon, Label } from '../src'
+
+const DropdownFlex = Flex.extend`
+  cursor: pointer;
+  height: 48px;
```

Comment:
Should this be `min-height`? What happens when text wraps?

Reply:
Nice feedback. Updated to use `min-height` to account for long label name that wraps.